tajo-core/src/main/java/org/apache/tajo/engine/planner/global/GlobalPlanner.java

```diff
@@ -1130,6 +1130,18 @@ private void setShuffleKeysFromPartitionedTableStore(StoreTableNode node, DataCh
       }
       channel.setShuffleKeys(shuffleKeys);
       channel.setShuffleType(SCATTERED_HASH_SHUFFLE);
+    } else if (node.getType() == NodeType.CREATE_TABLE) {
```

Comment:
The newly added code is the mostly same as those of INSERT.  It would be nice if some code to get ProjectSchema and InSchema are distinguished.